src/lib/wizard/components/Deploy.svelte

```diff
@@ -415,7 +417,9 @@
     <div class="flex flex-row items-center gap-2">
       <i class={`fa fa-lightbulb-o text-lime-600`}></i>
       <div class="text-xs text-gray-600">
-        <p>Ensure you have an Explorer API Key set in your <u><a href='{deploymentEnvironmentUrlWithFallback}' target='_blank'>Deploy Environment</a></u> for the current network to allow the contract to be verified automatically.</p>
+        {#if !hasSetUpBlockExplorerKeyForCurrentNetwork}
+          <p>Ensure you have an Explorer API Key set in your <u><a href='{deploymentEnvironmentUrlWithFallback}' target='_blank'>Deploy Environment</a></u> for the current network to allow the contract to be verified automatically.</p>
+        {/if}
         <p class="mt-2">Or download the <button type="button" onclick={downloadSolcInputHandler}><u>Solidity standard input JSON</u></button> for this contract to verify it manually on block explorers, using Solidity compiler version <code>{solcVersion}</code>.</p>
```

Comment:
We can also show a message if there is a key, otherwise the next paragraph sounds out of place.
```suggestion
        {:else}
          <p>The contract will be verified automatically on the block explorer for this network.</p>
        {/if}
        <p class="mt-2">Or you can download the <button type="button" onclick={downloadSolcInputHandler}><u>Solidity standard input JSON</u></button> for this contract to verify it manually on block explorers, using Solidity compiler version <code>{solcVersion}</code>.</p>
```